lib/data/COUNTY_NAME_TO_CODE.yml

```diff
@@ -1627,6 +1631,7 @@
     WEBSTER: 225
     WORTH: 227
     WRIGHT: 229
+    ST. LOUIS CITY: 510
```

Comment:
will we need a non-dot version?
ST LOUIS CITY: 510 ?

Reply:
I haven't seen that anywhere, name presentation data that I have seen includes the dot consistently